app/javascript/packs/dashboard/components/Dashboard.vue

```diff
@@ -1,13 +1,7 @@
 <template>
   <v-app>
-    <v-container style="max-width: 100%">
-      <v-carousel style="height: 250px">
-        <v-carousel-item
-          v-for="(item,i) in items"
-          :key="i"
-          :src="item.src"
-        ></v-carousel-item>
-      </v-carousel>
+    <v-container fluid>
+      <trail isActive isFavorite title="Trilha de introdução"></trail>
```

Comment:
Here we don't need to pass the value `true` to `isActive` props?

Reply:
Not necessarily. In Vue, boolean props can be used as flags. 🔝